src/Image/ImageDimensions.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Image;
+
+use Imagine\Image\BoxInterface;
+
+/**
+ * Image Dimensions
+ *
+ * @author Martin Auswöger <martin@auswoeger.com>
+ */
+class ImageDimensions
+{
+    /**
+     * @var BoxInterface
+     */
+    private $size;
+
+    /**
+     * @var bool
+     */
+    private $relative;
+
+    /**
+     * Constructor.
+     *
+     * @param BoxInterface $size     The size
+     * @param bool         $relative The relative flag
+     */
+    public function __construct(
+        BoxInterface $size,
+        $relative = false
+    ) {
+        $this->size = $size;
+        $this->relative = (bool) $relative;
```

Comment:
For images with a ratio, but no absolute size. E.g. a SVG image with `width="100%" height="100%"`.
See the difference between `File->imageSize` and `File->imageViewSize` in contao/core#7882.